docs-archive/apache-airflow/2.6.2/_api/airflow/decorators/index.html

```diff
@@ -806,8 +806,8 @@ <h3>Functions<a class="headerlink" href="#functions" title="Permalink to this he
 <tr class="row-even"><td><p><code class="xref py py-obj docutils literal notranslate"><span class="pre">short_circuit_task</span></code>([python_callable, multiple_outputs])</p></td>
 <td><p>Wraps a function into an ShortCircuitOperator.</p></td>
 </tr>
-<tr class="row-odd"><td><p><a class="reference internal" href="task_group/index.html#module-airflow.decorators.task_group" title="airflow.decorators.task_group"><code class="xref py py-obj docutils literal notranslate"><span class="pre">task_group</span></code></a></p></td>
-<td><p>Implements the <code class="docutils literal notranslate"><span class="pre">&#64;task_group</span></code> function decorator.</p></td>
+<tr class="row-odd"><td><p><a class="reference internal" href="task_group/index.html#module-airflow.decorators.task_group" title="airflow.decorators.task_group"><code class="xref py py-obj docutils literal notranslate"><span class="pre">task_group</span></code></a>(…)</p></td>
+<td><p>Python TaskGroup decorator.</p></td>
```

Comment:
was this suppose to change?

Reply:
Changes the task group line, not sure why.

![image](https://github.com/apache/airflow-site/assets/14861206/4ec68f8d-55fd-4b31-8c74-b5ecef64eafa)
![image](https://github.com/apache/airflow-site/assets/14861206/7db61616-1961-435c-a511-b32a10610f34)

Reply:
Not sure why we have this change here, maybe because of sphinx version mismatch

Reply:
interesting but anyone we can still merge this one

Reply:
Did it again after upgrading, we still have this change